stateful-workload-filestore/writer-fs.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_stateful_writer]
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: writer
+spec:
+  replicas: 2 # start with 2 replicas
+  selector:
+    matchLabels:
+      app: writer
+  template:
+    metadata:
+      labels:
+        app: writer
+    spec:
+      containers:
+      - name: content
+        image: us-docker.pkg.dev/google-samples/containers/gke/stateful-workload:latest
+        volumeMounts:
+        - name: fileserver
+          mountPath: /html # the shared directory
+        command: ["/bin/sh", "-c"]
+        args:
+        - cp /htmlTemp/indexInfo.html /html/index.html;
+          while true; do
+          echo "<b> Date :</b> <text>$(date)</text> <b> Writer :</b> <text2> ${HOSTNAME} </text2> <br>  " >> /html/indexData.html;
+          sleep 30;  
+          done
```

Comment:
Would it make more sense to have this in a `.sh` file and execute that script from here?